0.0.38 cannot be installed using npm 8

When running `npm i` on openapix 0.0.38 using npm 8, the following error occurs:

```
npm ERR! code EOVERRIDE
npm ERR! Override for @types/prettier@v2.6.0 conflicts with direct dependency
```

This appears to be due to `@types/prettier` being defined as `v2.6.0` in `devDependencies` and the override for `@types/prettier` being `2.6.0`.

--- a/.projenrc.js
+++ b/.projenrc.js
@@ -34,7 +34,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     '@types/lodash',
     '@types/js-yaml',
     '@types/omit-deep-lodash',
-    '@types/prettier@v2.6.0',
+    '@types/prettier@2.6.0',
     '@types/semver',
   ],
   bundledDeps: [

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "@types/lodash": "^4.14.184",
     "@types/node": "^14",
     "@types/omit-deep-lodash": "^1.1.1",
-    "@types/prettier": "v2.6.0",
+    "@types/prettier": "2.6.0",
     "@types/semver": "^7.3.12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",

